docs/bench/sqlite3-parser-comparison.md

```
@@ -0,0 +1,69 @@
+This report runs the sqlite3-parser-js-style input cases with `rawsql-ts` and `node-sql-parser` added for comparison.
+
+## Command
+
+```bash
+pnpm --filter rawsql-ts run benchmark:sqlite3-parser
+pnpm --filter rawsql-ts run benchmark:sqlite3-parser:bun
+```
+
+## Latest Run
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)